### DISCUSSION TITLE
Livewire + Alpine: closing a modal doesn't respect animations

### DISCUSSION THREAD

**Discussion by adddz** [livewire/livewire#1885] [1. Help] [answered]
### Description

I'm following Livewire screencasts where Caleb creates an Edit/Create modal (the modal is taken from Jetstream).
It works but there's something that is driving me crazy, when you click "Cancel" or "Save", the modal **abruptly closes** and skips the nice and smooth fade out animation that happens instead when you click on the backdrop behind the modal.

Also, the UX is not optimal when the network connection isn't as fast. The modal is not opened right away.
What would be the best practice to achieve the same modal but with a proper 'close/save' animation and snappiness?

### Stripped-down, copy-pastable code snippets

**Context**
- Livewire version: 2.2.3
- Laravel version: 8.3.1
- Alpine version: 2.3.0
- Browser: Chrome, Safari

**Comment by iksaku** [comment]
If you're concerned about reducing network requests for Modal open/close, use AlpineJS instead.

You can always fire a browser event from either your Livewire component or from Alpine itself, and in your modal listen for that event to open/close.

If you need a better example, you can check [MOPS modal implementation](https://github.com/iksaku/laravel-mops/blob/main/resources/views/components/modal.blade.php) using AlpineJS.

**Comment by joshhanley** [comment, marked as answer]
FYI this has been fixed in Livewire 2.3.1 and Alpine 2.7.3

**Reply by iksaku** [reply]
Please mark your comment as the “answer” to this issue, so people that come by also know that they need to update to newer versions 😉

**Reply by joshhanley** [reply]
@adddz If this is working for you after the update, can you mark as answered as @iksaku suggested, as sadly I cannot!
